assets/base.css

```diff
@@ -354,73 +354,73 @@ h5,
   font-family: var(--font-heading-family);
```

Comment:
When we have the font size range to `150%` the text gets cut off in the `collage` https://screenshot.click/28-03-89dbl-7ohm8.png

This happens when the scales are different: https://screenshot.click/28-13-47dfl-m832n.png

Reply:
Would prefer to address this in a separate PR. The reason being that this would require some refactoring and re-testing of the collage section. To be clear, this issue already exists with collage since longer product/collections titles could always cause overflow, but you are correct that this issue is made worse by the option to increase font scale

Reply:
Since the issue already exists, we can do that. Thanks for creating the issue :)